ports/stm32/Makefile

```diff
@@ -22,7 +22,12 @@ QSTR_DEFS = qstrdefsport.h $(QSTR_GENERATED_HEADERS)
 QSTR_GLOBAL_DEPENDENCIES = mpconfigboard_common.h $(BOARD_DIR)/mpconfigboard.h $(QSTR_GENERATED_HEADERS)
 
 # MicroPython feature configurations
+ifeq ($(DEBUG), 1)
+# Disable text compression in debug builds
+MICROPY_ROM_TEXT_COMPRESSION = 0
```

Comment:
IMO it'd be cleaner to put this line down in the existing DEBUG section with the other DEBUG flags.  Then it keeps debugging features localised to one location.

Reply:
@dpgeorge Yes but I thought it may be confusing to override this flag somewhere else, going to update it.

Reply:
Since it uses `?=` here, that's like the default setting, which is expected be overridden (eg by a board).